rpcq/_utils.py

```diff
@@ -41,33 +45,41 @@ def rpc_request(method_name: str, *args, **kwargs) -> rpcq.messages.RPCRequest:
     )
 
 
-def rpc_reply(id: Union[str, int], result: Optional[object]) -> rpcq.messages.RPCReply:
+def rpc_reply(id: Union[str, int], result: Optional[object],
+              warnings: List[Warning] = None) -> rpcq.messages.RPCReply:
```

Comment:
:memo: being persnickety the type annotation is `warnings: Optional[List[Warning]] = None`